pkg/envoy/lds/listener.go

```diff
@@ -20,13 +20,15 @@ import (
 const (
 	inboundListenerName           = "inbound-listener"
 	outboundListenerName          = "outbound-listener"
+	multiclusterListenerName      = "multicluster-listener"
 	prometheusListenerName        = "inbound-prometheus-listener"
 	outboundEgressFilterChainName = "outbound-egress-filter-chain"
 	egressTCPProxyStatPrefix      = "egress-tcp-proxy"
 	singleIpv4Mask                = 32
 )
 
 func (lb *listenerBuilder) newOutboundListener() (*xds_listener.Listener, error) {
+	// TODO(steeling): I believe there only needs to be 2 filter chains, 1 HTTP, 1 tcp... not relevant to this PR.
```

Comment:
If you believe the existing code is not correct, would you mind following an issue and following up instead of dropping a TODO here.

Reply:
oops.. that was a personal note as i was writing the other envoy doc. removed now